compel/arch/x86/src/lib/infect.c

```diff
@@ -310,6 +315,8 @@ static int corrupt_extregs(pid_t pid)
 	pr_err("Corrupting %s for %d, seed %u\n", use_xsave ? "xsave" : "fpuregs", pid, init_seed);
 
 	if (!use_xsave) {
+		clean_mxcsr(&ext_regs);
```

Comment:
wow. Thanks for detailed explanation! I will experiment with this mask tomorrow. This is definitely more elegant way to fix the problem than just nullifying the mask.

Reply:
@0x7f454c46 I've reworked to use `0x0000ffbf` mask over the random garbage. Let's take a look how it will work on CI. I've tested this on the different machines which I have and it works perfectly.

Reply:
@mihalicyn the result LGTM, thanks!